core/src/main/java/io/aiven/klaw/service/DefaultDataService.java

```diff
@@ -214,17 +242,36 @@ public List<KwProperties> createDefaultProperties(int tenantId, String mailId) {
             "Enable View or retrieve schemas");
     kwPropertiesList.add(kwProperties20);
 
-    KwProperties kwProperties21 =
-        new KwProperties(
-            "klaw.clusterapi.url", tenantId, KwConstants.CLUSTERAPI_URL, "Cluster Api URL");
+    KwProperties kwProperties21;
+
```

Comment:
The only thing I might do here is split the two into a method so kwProperties21 and kwProperties22 only have 1 if and you can see all the pieces that are applicable to the quickstart in one place.

Reply:
Updated.